apps/sms/views/conversation/test/unit/compose_test.js

```diff
@@ -1127,16 +1127,20 @@ suite('compose_test.js', function() {
         assert.isTrue(sendButton.disabled);
       });
 
-      test('enabled if both lock and Compose allow sending', function() {
+      test('enabled if lock allow sending', function() {
         Compose.setupLock({ canSend: () => true });
         Compose.refresh();
 
-        assert.isTrue(sendButton.disabled);
+        assert.isFalse(sendButton.disabled);
 
         Compose.append('Hola');
         assert.isFalse(sendButton.disabled);
       });
 
+      test('enabled when there is no message input', function() {
```

Comment:
__nit:__ Honestly I don't see value in both this test and the test below (```enabled when there is message input```) - it's already verified in the test above - so I think we can safely remove these two tests.